app/src/main/res/layout/sheet_bmi_reading.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we have used Compose for new layouts/features?

Reply:
It was much more work to implement using compose. As we already had couple of bottom sheet with same features it took less effort to do it using xml. Maybe it's best when we revamp all the bottom sheets at once. Would be a less effort then